README.md

```diff
@@ -90,50 +95,66 @@ When you run rubric tester, the datasets and experiments you use will be copied
 To set up rubric tester to run locally:
 
 install pyenv: https://github.com/pyenv/pyenv?tab=readme-ov-file#installation
-* Mac: `brew install pyenv`
-* Ubuntu: `curl https://pyenv.run | bash`
+
+- Mac: `brew install pyenv`
+- Ubuntu: `curl https://pyenv.run | bash`
 
 install python 3.11:
-* `pyenv install 3.11.7`
+
+- `pyenv install 3.11.7`
 
 set python 3.11 for the aiproxy repo:
-* `cd aiproxy`
-* `pyenv local 3.11.7`
+
+- `cd aiproxy`
+- `pyenv local 3.11.7`
 
 create a python virtual environment at the top of the directory:
-* `python -m venv .venv`
+
+- `python -m venv .venv`
 
 ensure aws access for accessing aws bedrock models:
-* from the code-dot-org repo root, run:
-  * `bin/aws_access`
-* from this repo's root, run: 
-  * `gem install aws-google`
-  
+
+- from the code-dot-org repo root, run:
+  - `bin/aws_access`
+- from this repo's root, run:
+  - `gem install aws-google`
+
+Clone the aitt_release_data repo (https://github.com/code-dot-org/aitt_release_data). (Note: `rubric_tester` will look for this in the folder you cloned aiproxy into and will attempt to clone it in that location if it cannot locate it).
```

Comment:
what's the advantage of including cloning as a manual step if rubric tester takes care of it automatically?

Reply:
Good question